packages/salesforcedx-utils-vscode/test/jest/helpers/paths.test.ts

```diff
@@ -70,4 +71,39 @@ describe('test project paths', () => {
       expect(relativeStateFolder).toEqual(FAKE_STATE_FOLDER);
     });
   });
+
+  describe('test SFDXGlobalConfigPath', () => {
+    it('should return the global config path', () => {
+      const result = projectPaths.sfdxGlobalConfig();
+      expect(result).toEqual(path.join(os.homedir(), '.sfdx', 'sfdx-config.json')
+      );
+    });
+  });
+
+  describe('test SFDXConfigPath', () => {
+    const FAKE_CONFIG = path.join(FAKE_WORKSPACE, 'sfdx-config.json');
```

Comment:
I think the state folder is missing here, should be: `(FAKE_WORKSPACE, FAKE_STATE_FOLDER, FAKE_CONFIG_FILE)`